src/stream.ts

```diff
@@ -168,6 +179,9 @@ export class WebRTCStream implements Stream {
     this.readClosed = true;
     this.writeClosed = true;
     this.channel.close();
+    if (this.closeCb) {
```

Comment:
I had to think for a second about what cb might stand for. Perhaps the name could be more verbose.